src/components/UserList.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState, useContext } from 'react';
+import BeatLoader from 'react-spinners/BeatLoader';
+import { UserContext } from '../contexts/UserContext';
+
+export default function UserList({ userList }) {
+  const [users, setUsers] = useState([]);
+  const [isLoading, setIsLoading] = useState();
+  const { setUser } = useContext(UserContext);
+  const [selectedUser, setSelectedUser] = useState();
+
+  useEffect(() => {
+    setIsLoading(true);
+    setUsers(userList.users);
```

Comment:
There's some duplication of state going on here I believe? We have userList as a state in the parent component but then also passed to the users state locally to this component. Is there a reason for this as it seems unnecessary to me?

Reply:
There's some duplication of state going on here I believe? We have userList as a state in the parent component but then also passed to the users state locally to this component. Is there a reason for this as it seems unnecessary to me?